### ISSUE TITLE
"85 - braces_alpha_range1" fails on editorconfig-core-c

### ISSUE THREAD

**Issue by ppalaga** [editorconfig/editorconfig-core-test#13]
https://github.com/editorconfig/editorconfig-core-c being the canonical implementation, one expects all core tests to pass there. However, this is not the case. "85 - braces_alpha_range1" as present in rev 172eb8324225f09f3e331af26454a5b45e314edb of core-tests is failing on core-c rev 6f3c40d510068935322e5e25c2196e12584cca04

Steps to reproduce:

```
# cd where editorconfig-core-c was cloned
cd editorconfig-core-c
git submodule init
git submodule update
cd tests
git status
HEAD detached at efc9b44 # quite an old revision
git checkout 172eb8324225f09f3e331af26454a5b45e314edb # currently the master's HEAD
cd ..
cmake .
ctest .
...
# Actual:
The following tests FAILED:
         85 - braces_alpha_range1 (Failed)

# Expected all tests should pass
```

The test is defined here: https://github.com/editorconfig/editorconfig-core-test/blob/172eb8324225f09f3e331af26454a5b45e314edb/glob/CMakeLists.txt#L204
When looking at the test, it does not seem to make any sense. We pass file name `{aardvark..antimater}` and we expect it to match glob `[{aardvark..antelope}]` and return the proprety `words=a`. 

Well, given that core-c is has no support for alphanumeric ranges, the test should either 

(a) expect no property to get returned

or 

(b) it should pass `{aardvark..antelope}` as a file name so that the `[{aardvark..antelope}]` glob matches. 

Or perhaps 

(c) if core-c had a support for alpha ranges, the file name should be something from the range, e.g. `aardvark`.

I am ready to submit a RP once somebody decides which of the options is favorable.

**Comment by ppalaga** [comment]
cc @angelozerr

**Comment by xuhdev** [comment]
@ppalaga Are you sure the test passes for you before the change?

**Comment by xuhdev** [comment]
I looked into this again -- I think this was probably a misspell: there is really no support for "letter range", and the fact that it never failed before is caused by two overlapped testing name in previous test cases. Any way, I'm going to fix this in a few minutes.

**Comment by xuhdev** [comment]
Fixed in 552ccb7a6128a8459f6a3aec80f51a980469c9fc

**Comment by ppalaga** [comment]
@xuhdev thanks!
